src/task21.java

```diff
@@ -0,0 +1,18 @@
+public class task21 {
+
+        public static void main(String[] args) {
+            String s = "How are you?";
```

Comment:
"How are , you?"
Exception in thread "main" java.lang.StringIndexOutOfBoundsException: String index out of range: -1